tests/installation/first_boot.pm

```diff
@@ -15,6 +15,7 @@
 
 use strict;
 use base "y2logsstep";
+use bootbasetest;
```

Comment:
@DrMullings Looking at this again... changing this line to `use base "bootbasetest'` and moving it above y2logsstep should do the trick too. (use base qw(bootbasetest y2logsstep) should also do the trick and call the post_fail_hook from bootbasetest.